tests/unit/amo/sagas/test_addonsByAuthors.js

```diff
@@ -50,6 +50,8 @@ describe(__filename, () => {
   }
 
   it('calls the API with addonType if set', async () => {
+    const fakeConfig = getFakeConfig({ enableStaticThemes: false });
+    const root = render({ _config: fakeConfig };
```

Comment:
actually in this test case you can remove this and add the following where you see addonType here:

`addonType: getAddonTypeFilter(ADDON_TYPE_THEME)`

you will need to import this function (getAddonTypeFilter) too at the top of the file:
import { getAddonTypeFilter } from 'core/utils';